docs/UserGuide.md

```diff
@@ -9,102 +9,197 @@ title: User Guide
+| Field                 | A field is the information following the slash in a command.                                                                                                |
+| Medical Qualification | The level of care taking or first aid of a volunteer. It consists of the type of skill (E.g. CP, AED) and a skill level (`BASIC`, `INTERMEDIATE` or `HIGH`) |
+| NRIC                  | A unique identifier given to all Singaporeans. It is case-insensitive.                                                                                      |
+| Pair                  | A pair consists of an elderly and a volunteer assigned to accompany and take care of the elderly                                                            | 
+| Phone number          | The phone number of a person. Must be numeric and has more than 3 digits                                                                                    |
+| Prefixes              | Prefixes are the characters appearing before a slash in a command. Prefixes describe the attribute that they represent.                                     |
+| Region                | The general portion of area in Singapore. Must be one of the following values: `NORTH`, `NORTHEAST`, `CENTRAL`, `WEST` or `EAST`                            |
+| Risk level            | The susceptibility level of an elderly to injury or sickness. Must be one of the following values: `LOW`, `MEDIUM` or `HIGH`                                |
+| Tag                   | A generic description for a group of people. Must contain only alphanumeric characters                                                                      |
+| Volunteer             | Volunteers that signed up to pair up with and accompany elderly members                                                                                     |
+| VWO                   | Voluntary Welfare Organisations such as yourself                                                                                                            |
 
 ## Quick Start
 
-1. Ensure you have Java `11` or above installed in your Computer.
-
-2. Download the latest `friendlyLink.jar` from [here](https://github.com/AY2223S2-CS2103T-W12-1/tp/releases).
+1. Ensure you have [Java 11](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html) installed in your Computer.
 
-3. Copy the file to an empty folder you want to use as the _home folder_ for your FriendlyLink.
+1. Download the latest `friendlylink.jar` from [here](https://github.com/AY2223S2-CS2103T-W12-1/tp/releases).
 
-4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar friendlylink.jar` command to run the application.
-   The GUI should appear in a few seconds. <br>
+1. Move the `friendlylink.jar` to an empty folder where you want FriendlyLink to store information.
```

Comment:
```suggestion
2. Move the `friendlylink.jar` to an empty folder where you want FriendlyLink to store information.
```

Reply:
I think we can use github auto numbering for this, so we will leave all as 1.